packages/@aws-cdk/aws-stepfunctions-tasks/lib/sagemaker/create-transform-job.ts

```diff
@@ -226,10 +226,11 @@ export class SageMakerCreateTransformJob extends sfn.TaskStateBase {
   }
 
   private renderTransformResources(resources: TransformResources): { [key: string]: any } {
+    let instanceType = resources.instanceType instanceof ec2.InstanceType ? 'ml.' + resources.instanceType : resources.instanceType;
```

Comment:
why not `const`? it's not being re-assigned anywhere.

also: use string interpolation instead of concatenation.

Reply:
Thanks, fixed